source/solvers/heat_transfer_assemblers.cc

```diff
@@ -60,43 +60,46 @@ HeatTransferAssemblerCore<dim>::assemble_matrix(
       // parameter
       const double u_mag = std::max(velocity.norm(), 1e-12);
 
-      // Temperature gradient information for DCDD stabilization
-      Tensor<1, dim> dcdd_temperature_gradient =
-        is_steady(method) ? scratch_data.temperature_gradients[q] :
-                            scratch_data.previous_temperature_gradients[0][q];
-
-      // Implementation of a Discontinuity-Capturing Directional Dissipation
-      // (DCDD) shock capturing scheme. For more information see Tezduyar,
-      // T. E. (2003). Computation of moving boundaries and interfaces and
-      // stabilization parameters. International Journal for Numerical
-      // Methods in Fluids, 43(5), 555-575. Our implementation is based on
-      // equations (70) and (79), which are adapted for the heat transfer
-      // solver.
-
-      // In Tezduyar 2003, this is denoted r in equation (67). The velocity is
-      // replaced by the temperature.
-      Tensor<1, dim> temperature_gradient_unit_vector =
-        dcdd_temperature_gradient / (dcdd_temperature_gradient.norm() + 1e-12);
-
-      // In Tezduyar 2003, this is denoted s in equation (67)
-      Tensor<1, dim> velocity_unit_vector =
-        velocity / (velocity.norm() + 1e-12);
-
-      // Directionality tensor describing [rr - (r⋅s)^2*ss]
-      Tensor<2, dim> dir_tensor =
-        outer_product(temperature_gradient_unit_vector,
-                      temperature_gradient_unit_vector) -
-        (Utilities::fixed_power<2>(scalar_product(
-           temperature_gradient_unit_vector, velocity_unit_vector)) *
-         outer_product(velocity_unit_vector, velocity_unit_vector));
-
-      // Calculate the artificial viscosity of the shock capture as in equation
-      // (79) of Tezduyar 2003
-      const double nu_dcdd = is_steady(method) ?
-                               0 :
-                               0.5 * h * h * u_mag *
-                                 dcdd_temperature_gradient.norm() /
-                                 scratch_data.global_delta_T_ref;
+      // // Temperature gradient information for DCDD stabilization
+      // Tensor<1, dim> dcdd_temperature_gradient =
+      //   is_steady(method) ? scratch_data.temperature_gradients[q] :
+      //                       scratch_data.previous_temperature_gradients[0][q];
+
+      // // Implementation of a Discontinuity-Capturing Directional Dissipation
+      // // (DCDD) shock capturing scheme. For more information see Tezduyar,
+      // // T. E. (2003). Computation of moving boundaries and interfaces and
+      // // stabilization parameters. International Journal for Numerical
+      // // Methods in Fluids, 43(5), 555-575. Our implementation is based on
+      // // equations (70) and (79), which are adapted for the heat transfer
+      // // solver.
+
+      // // In Tezduyar 2003, this is denoted r in equation (67). The velocity
```

Comment:
I would delete all these comments. No need to keep it. It's better your new imeplemetaitno.